tests/plugins/auto-sizes/auto-sizes-test.php

```diff
@@ -87,4 +87,16 @@ public function test_content_image_without_lazy_loading_does_not_have_auto_sizes
 			wp_filter_content_tags( $this->get_image_tag( self::$image_id ) )
 		);
 	}
+
+	/**
+	 * Test printing the meta generator tag.
+	 *
+	 * @covers ::auto_sizes_render_generator
+	 */
+	public function test_auto_sizes_render_generator() {
```

Comment:
I'd be for adding an additional test to ensure this callback is properly hooked to the `wp_head` action, similar to [this example](https://github.com/WordPress/performance/pull/1103#discussion_r1546577164).

Reply:
Fair point, we can add this here. Though then I'd say lets also assert the other hooks in the same test.